### PR TITLE
Add TAIKO ERC-20 token to Ethereum mainnet

### PR DESCRIPTION
**Pull request by cheeky-gorilla**
### Contribution type

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

- **Description:** Add TAIKO ERC-20 token to Ethereum mainnet

https://etherscan.io/address/0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800

I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Please sign the CLA @cheeky-gorilla so we can continue to merge this PR.

**Comment by fredriksvantes** [comment]
recheck

**Comment by cheeky-gorilla** [comment]
recheck

**Comment by cheeky-gorilla** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cheeky-gorilla** [comment]
recheck

**Comment by cheeky-gorilla** [comment]
Done @Hosuke :)

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 1]
```diff
@@ -4560,5 +4560,6 @@ FROM (VALUES
     , ( 0xfc87753df5ef5c368b5fba8d4c5043b77e8c5b39, 'aETH', 18)
     , ( 0xf1617882a71467534d14eee865922de1395c9e89, 'saETH', 18)
     , ( 0x9deb0fc809955b79c85e82918e8586d3b7d2695a, 'GOLD', 18)
-    , ( 0x75e88b8c2d34a52a6d36deada664d7dc9116e4ef, 'ZRS', 18)    
+    , ( 0x75e88b8c2d34a52a6d36deada664d7dc9116e4ef, 'ZRS', 18)
+    , ( 0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800, 'TAIKO', 18)
```
Comment (Hosuke) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:4564]: https://etherscan.io/token/0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800

**Review by Hosuke — APPROVED** [review]

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**